pkg/steampipeconfig/modconfig/parsed_name.go

```diff
@@ -55,7 +55,12 @@ func (p *ParsedResourceName) ToFullNameWithMod(mod string) string {
 }
 
 func BuildFullResourceName(mod, blockType, name string) string {
```

Comment:
why is this being called with an empty block type?  `mod.name` is not a valid resource name. 

I think this function shgould validate it's params but the calling code should possibly not use this function